.github/workflows/go.yml

```
@@ -22,5 +22,11 @@ jobs:
     - name: Check out code into the Go module directory
       uses: actions/checkout@v2
 
+    - name: Check tidy go modules
+      run: |-
+         [ -z "$(git status --porcelain)" ] || (echo "Dirty!" && exit 1)
+         make tidy
```

The first check will never occur because `action/checkout@v2` will always have a clean repo

```suggestion
         make fmt tidy
```